flatpak/README.md

```
@@ -0,0 +1,101 @@
+./flatpak/build.sh
+```
+
+### Manual Build
+```bash
+flatpak-builder --force-clean --repo=repo build-dir flatpak/org.lotti.Lotti.yml.template
+```
+
+## Installing
+
+### Install Locally
+```bash
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)